packages/terra-button-group/tests/nightwatch/ButtonGroupSingelSelect.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
Typo in file name: SingleSelect

Reply:
Updated:https://github.com/cerner/terra-core/pull/1155/commits/ed6ceb87b5bfd650e1259d62a29f077c6b4ef36f